### PR TITLE
replace init resource with quarkus

### PR DESCRIPTION
**Pull request by shokakucarrier**

### CODE REVIEW

**Comment by shokakucarrier** [comment]
@ligangty @yma88 can we merge this?

File: src/main/resources/application.xml [hunk 1]
```diff
@@ -1,55 +0,0 @@
...
-      level: INFO
-      enable: true
-      path: "log/indy-archive-service.log"
-      format: "%d{HH:mm:ss} %-5p [%c{2.}] (%t) %s%e%n"
-      rotation:
-        max-backup-index: 5
-        max-file-size: 10M
-
-  swagger-ui:
-    always-include: true
```
Comment (ligangty) [inline src/main/resources/application.xml:44]: I think this file should not be removed but just to rename to yml with your items merging

**Review by ligangty — COMMENTED** [review]
LGTM except a minor issue.

**Review by yma955 — APPROVED** [review]
LGTM

**Pull request merged**